pkg/types/aws/defaults/platform.go

```diff
@@ -6,13 +6,13 @@ import (
 
 var (
 	defaultMachineClass = map[string][]string{
-		"ap-east-1":      {"m5", "m4"},
-		"ap-northeast-2": {"m5", "m4"},
-		"eu-north-1":     {"m5", "m4"},
-		"eu-west-3":      {"m5", "m4"},
-		"me-south-1":     {"m5", "m4"},
-		"us-gov-east-1":  {"m5", "m4"},
-		"us-west-2":      {"m5", "m4"},
+		// "ap-east-1":      {"m5", "m4"},
```

Comment:
@abhinavdahiya Should we just rip out all the logic for having different defaults for different regions or keep the logic and structure in case we need to implement something similar in the future?

Reply:
i think we should remove the overrides, but keep the mechanism in place.

Reply:
we need to keep some overrides for regions/zones that lack m5 support, right?

Reply:
never mind, I'm stale after the removal of `platformtests/` (which I've just filed #3856 to wrap up).  I'm not clear on why we would need/want the ability to set overrides if we trust the dynamic APIs we've used since #3051.